mm/mm_heap/mm_initialize.c

```diff
@@ -97,7 +97,8 @@ void mm_addregion(FAR struct mm_heap_s *heap, FAR void *heapstart,
    * with the MM_MIN_CHUNK size.
```

Comment:
let's copy the comment from https://github.com/apache/incubator-nuttx/pull/5991

Reply:
Done